contentcuration/contentcuration/assets/shared/views/ThumbnailUpload.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <div
+    :style="highlight && !readonly ? { borderColor: 'purple' } : {}"
+    @dragenter.prevent="enter"
+    @dragover.prevent="over"
+    @dragleave.prevent="leave"
+    @drop.prevent="drop"
+  >
+    <template
+      v-for="file in files"
+    >
+      <VProgressCircular
+        v-if="file.progress < 100"
+        :key="file.id"
+        :size="50"
+        :width="15"
+        :value="file.progress"
+      >
+        {{ file.progress }}
+      </VProgressCircular>
+    </template>
+    <input
+      v-if="!readonly"
+      id="fileElem"
+      ref="fileUpload"
+      style="display: none;"
+      type="file"
+      accept="image/*"
```

Comment:
Might need to explicitly list out jpg and png here as we don't support gifs or svgs as thumbnails (or we might want to change that restriction instead)